packages/target-decisioning-engine/src/contextProvider.js

```diff
@@ -121,10 +122,11 @@ function createNestedParametersFromDots(context) {
   let result = {};
   Object.keys(context).forEach(key => {
     if (
-      key.indexOf(".") > -1 &&
-      key.indexOf("..") === -1 &&
+      includes(".", key) &&
+      !includes("..", key) &&
       key[0] !== "." &&
-      key[key.length - 1] !== "."
+      !key.endsWith(".") &&
+      !key.endsWith("._lc")
```

Comment:
@jasonwaters realize this is a bit hardcoded but thought it was better than changing `getLowerCaseAttributes` as it is used twice or copying `getLowerCaseAttributes` functionality into `createNestedParametersFromDots`